base/ltcmdhooks.dtx

```diff
@@ -12,16 +12,20 @@
 %    https://www.latex-project.org/lppl.txt
 %
 %
-%%% From File: ltcmdhooks.dtx
-%
-\def\ltcmdhooksversion{v1.0i}
-\def\ltcmdhooksdate{2023/11/28}
-%
+% \fi
 %
+% \iffalse
+%%% From File: lthooks.dtx
 %
 %<*driver>
+% \fi
+\ProvidesFile{ltcmdhooks.dtx}
+             [2024/02/11 v1.0i LaTeX Kernel (Command hooks)]
+% \iffalse
+%
 \documentclass{l3doc}
-%\usepackage{ltcmdhooks}
+\GetFileInfo{ltcmdhooks.dtx}
+% \usepackage{ltcmdhooks}
```

Comment:
why not throw it out?

Reply:
the commented out line I mean

Reply:
yes I left it in to keep the diff focussed on the issue rather than rolling in unrelated cleanup, but it can certainly go.